modules/monitoring/library/Monitoring/Web/Controller/MonitoredObjectController.php

```diff
@@ -160,6 +163,8 @@ protected function handleFormatRequest($query = null)
                 ->setSuccessData($payload)
                 ->setAutoSanitize()
                 ->sendResponse();
+        } else {
+            parent::handleFormatRequest($query);
```

Comment:
Please change this so that the base implementation is called everytime `$query` is passed. At the moment we're checking the parameters without need and don't provide a output for the given query if they match.

```php
if ($query !== null) {
    parent::handleFormatRequest($query);
} elseif ($this->params->...
```